Updated the screenfull package to 5.0.0 version

intity opened the pull request:
Fixed the issue #32

intity committed:
Updated the screenfull package to 5.0.0 version

intity committed:
Update deps

intity committed:
Added the upload book from PC feature

intity committed:
Fixed the restore option for default epub reader

intity committed:
added vscode directory to .gitignore

intity committed:
updated packages

maggie44 commented:
This resolves the full-screen issue, and brings the EPUBJS up to date which also has a bunch of fixes. 

@intity it seems unlikely this is going to have continued maintenance, since the pull request is over a year old. Might be time to think about maintaining a separate repo of this code ourselves, I have a bunch of other things to create pull requests for if you want to start it? 

Flagging @fchasen here in case of any objections to avoid duplication.

intity commented:
Unfortunately I don't have much time to develop epubjs-reader. Initially, my goal was to create a simple [chrome browser extension](https://github.com/intity/epubreader-chrome) to upload books in epub format by URL or from local storage.

intity committed:
added .editorconfig file

maggie44 commented:
> Unfortunately I don't have much time to develop epubjs-reader. Initially, my goal was to create a simple [chrome browser extension](https://github.com/intity/epubreader-chrome) to upload books in epub format by URL or from local storage.

Me neither really, but was thinking making it more of a maintenance mode, confirming pull requests and merging. Seems like most of that is done by the community just not merged, and the development is still very active in the epubjs repo which sits at its core. 

Hopefully @fchasen will get back on it, and if there are no plans to continue this repo then will look at a forked alternative instead.

intity committed:
updated dependency packages

intity committed:
refactoring UI elements

intity committed:
Merge branch 'upload_feature'

# Conflicts:
#	package-lock.json
#	reader/js/reader.js
#	reader/js/reader.js.map
#	reader/js/reader.min.js

intity committed:
updated package-lock.json

intity committed:
update builds

intity committed:
combine all css to main.css

intity committed:
added main.js

intity committed:
updated dependency packages

intity committed:
added storage feature

intity committed:
replaced copy jszip.min.js from jszip package

intity committed:
moved epub.min.js to libs directory

intity committed:
removed screenfull.min.js

intity committed:
removed reader.min.map

intity committed:
removed localforage.min.js

intity committed:
update package

intity committed:
added index.html in root package for debug

intity committed:
changed root directory to serve.js

intity committed:
fixed link to screenfull.js

intity committed:
fixed link to reader.min.js

intity committed:
fixed initialize default instance

intity committed:
added clearing storage before adding a new blob

intity committed:
moved upload controller to main

intity committed:
added copy main.js to Gruntfile

intity committed:
removed script upload_controller.js from index.html

intity committed:
updated .editorconfig

intity committed:
removed grunt-concat-sourcemap package

intity committed:
include rsvp.min.js as an external dependency

intity committed:
concatenated the storage.js with reader.js

intity committed:
added screenfull.js to uglify

intity committed:
core.js code verification

intity committed:
reader.js code verification

intity committed:
updated packages

intity committed:
updated builds

intity committed:
fixed error of undefined start property on page reload

intity committed:
refactoring the storage.set function

intity committed:
updated builds

intity committed:
replaced grunt build system with webpack

intity committed:
migrated the source to ES6 standard

intity committed:
incremental work:

- removed the jquery dependency
- removed the rsvp dependency
- added the signals dependency
- added the js-md5 dependency
- refactoring the styles
- refactoring the icons
- refactoring the epubjs-reader

intity committed:
updated packages

intity committed:
added a simple search throughout the contents of the book

intity committed:
temporarily disabling the reflowTextRow

intity committed:
update packages

intity committed:
updated builds

intity committed:
removed the main.js from build

intity committed:
removed the jszip library

intity committed:
removed the mime-types library

intity committed:
changed the output directory for build

intity committed:
moved build

intity committed:
cleanup the output directory

intity committed:
removed the optimist package

intity committed:
removed the xmldom package

intity committed:
change location if previous URL includes the bookPath token

intity committed:
refactoring the main

intity committed:
updated build

intity committed:
added the module type attribute

intity committed:
combine origin and pathname locations

intity committed:
Update README.md

the description of the build system has been changed

Commit:
Translation: French

Commit:
Add French to the language selection menu

intity committed:
Merge pull request #1 from vvillenave/vv/l10n-fr

Translation: French

intity committed:
updated packages

intity committed:
updated build

intity committed:
added the webpack-dev-server package

intity committed:
Update README.md

intity committed:
fixed pre-init the book settings

intity committed:
Merge branch 'master' of https://github.com/intity/epubjs-reader

intity committed:
updated build

intity committed:
fixed sidebar tabs translation

intity committed:
fixed search placeholder translate

intity committed:
updated build

kurema committed:
Update strings.js

kurema committed:
Update settings_panel.js

intity committed:
Merge pull request #2 from kurema/patch-1

Translation: Japanese

intity committed:
updated packages

intity committed:
updated build

intity committed:
fixed button height

intity committed:
updating packages

intity committed:
fixed preliminary language initialization

intity committed:
refactoring UIInteger class

intity committed:
added font size setting

intity committed:
cleanup

intity committed:
updating builds

intity committed:
fixed restore font size on reload browser

intity committed:
updating build

intity committed:
fixed font size sync for all controls

intity committed:
refactoring navigation controls

intity committed:
refactoring the Content class

intity committed:
updating packages

intity committed:
updating packages

intity committed:
updating packages

intity committed:
refactoring webpack configuration

intity committed:
importing epubjs modules into an epubreader library

intity committed:
added wheel event listener for disable zoom text

intity committed:
source cleaning

intity committed:
updating builds

intity committed:
rename the reader folder to dist

intity committed:
added the minify and prepare commands

intity committed:
updating packages

intity committed:
cleanup dependencies

intity committed:
removed ignore class definition

intity committed:
updating builds

intity committed:
updating package-lock

intity committed:
removed signals package

intity committed:
updating builds

intity committed:
removed link to signals library

intity committed:
fixed the location history state

intity committed:
added search url to redirection

intity committed:
added important rule to sidebar styles

intity committed:
removed the display flex property from viewer style

intity committed:
re-including the epub.js library as external (regression)

intity committed:
updating builds

intity committed:
updating builds

intity committed:
updating packages

intity committed:
updating libs

intity committed:
added the tree view ui

intity committed:
refactoring the TocPanel class

intity committed:
fixed redirect to cfi location

intity committed:
added the contents (TOC) selector

intity committed:
updating builds

intity committed:
updating packages

intity committed:
added Russian language to strings

intity committed:
updating builds

intity committed:
moved panels to root src

intity committed:
refactoring the AnnotationsPanel class

intity committed:
refactoring the BookmarksPanel class

intity committed:
refactoring the TocPanel class

intity committed:
refactoring the SearchPanel class

intity committed:
refactoring the SettingsPanel class

intity committed:
refactoring the MetadataPanel class

intity committed:
updating packages

intity committed:
updating builds

intity committed:
enabled default devtool as 'eval-source-map'

intity committed:
include source no module type definition

intity committed:
updating builds

intity committed:
fixed the license type to package.json

intity committed:
fixed the undefined location to rendition

intity committed:
added 'source-map' to all config

intity committed:
updating packages

intity committed:
updating builds

intity committed:
redefinition open book icon

intity committed:
refactoring sidebar icons

intity committed:
refactoring sidebar style

intity committed:
updating packages

intity committed:
updating builds

intity committed:
updating packages

intity committed:
updating packages
